docs/content_management/collaborative_editing/configure_collaborative_editing.md

```diff
@@ -0,0 +1,98 @@
+---
+description: Configure the Collaborative editing feature.
+month_change: true
+---
+
+# Collaborative editing
+
+Collaborative editing feature is available in [[= product_name =]] starting with version v5.0.2 or higher, regardless of its edition.
+
+## Installation
+
+### Install Real-time editing feature package
+
+If you have an arrangements with [[= product_name_base =]] to use Real-time editing feature, you need to install following package:
+
+``` bash
+composer require ibexa/fieldtype-richtext-rte
+```
+
+This command installs also `ibexa/ckeditor-premium` package and adds the new real-time editing functionality to the Rich Text field type.
+It also modifies the permission system to account for the new functionality.
+
+### Modify the bundles file
+
+Then, if not using Symfony Flex, add the following code to the `config/bundles.php` file:
+
+``` php
+<?php
+
+return [
+    // A lot of bundles…
+    Ibexa\Bundle\FieldTypeRichTextRTE\IbexaFieldTypeRichTextRTEBundle::class => ['all' => true],
+    Ibexa\Bundle\CkeditorPremium\IbexaCkeditorPremiumBundle::class => [‘all’ => true],
```

Comment:
Please compare the quotes - row with IbexaCkeditorPremiumBundle has the wrong ones (`'` vs `‘`). The difference is visible on preview

<img width="850" height="55" alt="Screenshot 2025-10-16 at 00 18 00" src="https://github.com/user-attachments/assets/9e779a8e-bc91-44fd-810f-36c78d258e35" />

It wouldn't work when copied from the doc to the app.